main/src/cgeo/geocaching/utils/UriUtils.java

```diff
@@ -22,10 +22,13 @@ public static String toUserDisplayableString(final Uri uri) {
         if (uri == null) {
             return "";
         }
-        final String uriString = uri.getPath();
+        String uriString = uri.getPath();
         final int idx = uriString.lastIndexOf(":");
         if (idx >= 0) {
-            return uriString.substring(idx + 1);
+            uriString = ".../" + uriString.substring(idx + 1);
```

Comment:
Please use the ellipsis character (`…`) instead of three dots. But anyway, @eddiemuc you are doing a great job!

Reply:
Will do that directly in storage_framework